grudge/op.py

```diff
@@ -478,13 +478,36 @@ def _cross_rank_trace_pairs_scalar_field(dcoll, vec, tag=None):
 
 
 def cross_rank_trace_pairs(dcoll, vec, tag=None):
```

Comment:
```suggestion
def cross_rank_trace_pairs(dcoll, ary, tag=None):
```
? (Given that `vec` is a bit of a misnomer now?)